kevm-pyk/src/tests/integration/test_foundry_prove.py

```diff
@@ -35,12 +35,15 @@ def foundry_root(tmp_path_factory: TempPathFactory) -> Path:
         includes=(),
         requires=[str(TEST_DATA_DIR / 'lemmas.k')],
         imports=['LoopsTest:SUM-TO-N-INVARIANT'],
+        llvm_library=use_booster,
     )
 
     return foundry_root
 
 
-def test_foundry_kompile(foundry_root: Path, update_expected_output: bool) -> None:
+def test_foundry_kompile(foundry_root: Path, update_expected_output: bool, use_booster: bool) -> None:
+    if use_booster:
+        return
```

Comment:
`use_booster` should only modify the RPC server started by the implementation. If you need more control on which checks are performed, consider creating a dedicated test module (e.g. `test_booster.py`).

Reply:
these are all necessary because the golden files will not exactly correspond to the output of the booster. when use_booster is enable all we are doing is testing that the tests pass.

Reply:
I see, makes sense.